test/automated_serial_generation_e2e_test.ts

```
@@ -5,6 +5,8 @@
 import * as path from "path";
 import type * as monaco from "monaco-editor";
 import { sleep } from "graphai";
+import { ElectronAPI } from "@/types/electron";
```

ESLint

[error] Remove this unused import of 'ElectronAPI'.

(sonarjs/unused-import)